achyut/src/main/java/com/lftechnology/runner/Menu.java

```diff
@@ -0,0 +1,22 @@
+package com.lftechnology.runner;
+
+public class Menu {
+
+    public Menu() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1186)